packages/dapp/src/components/QRCodeScannerStepper/ConnectDeviceView.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react';
+import { uint8ArrayToHex } from '@blockchain-lab-um/masca-connector';
+import { useTranslations } from 'next-intl';
+
+import { useGeneralStore, useSessionStore, useToastStore } from '@/stores';
+import Button from '../Button';
```

Comment:
Probably should use ˙@/components/Button` also change this in other places (and other components)